plugins/inventory/kubevirt.py

```diff
@@ -605,6 +619,22 @@ def get_vmis_for_namespace(
                 vmi_name, "vmi_volume_status", vmi_volume_status
             )
 
+            # Set up the connection
+            service = None
+            if self.is_windows(vmi_guest_os_info, vmi_annotations):
+                self.inventory.set_variable(vmi_name, "ansible_connection", "winrm")
```

Comment:
Should we want to hardcode `winrm` as the connection for all Windows instances? Windows does support SSH and it might be ideal to provide a way to override. Maybe based on annotation? What about other connection options that we might want to inject for the specific Virtual Machine?

Reply:
In the next released version with the `Constructable` feature fixed (see #77), you can do it with the `compose` key in the inventory plugin's configuration.

Example inventory configuration:

```
---
plugin: kubevirt.core.kubevirt
```

results in

```
ansible-inventory -i inventory.kubevirt.yml --list | jq -r '.["_meta"]["hostvars"]["default-testvm"]["ansible_connection"]'
ssh
```

Adding `compose`:

```
---
plugin: kubevirt.core.kubevirt
compose:
  ansible_connection: "'something'"
```

results in

```
ansible-inventory -i inventory.kubevirt.yml --list | jq -r '.["_meta"]["hostvars"]["default-testvm"]["ansible_connection"]'
something
```

Reply:
'compose' takes a Jinja template expression, so you can get quite creative. It should be able to override the value only for specific hosts too.

Reply:
That sounds great! Thanks for the update on the set of planned features